Tasks/Maven/maventask.ts

```diff
@@ -31,7 +32,8 @@ var isCodeCoverageOpted = (typeof ccTool != "undefined" && ccTool && ccTool.toLo
 let buildOutput: BuildOutput = new BuildOutput(tl.getVariable('build.sourcesDirectory'), BuildEngine.Maven);
 var codeAnalysisOrchestrator:CodeAnalysisOrchestrator = new CodeAnalysisOrchestrator(
     [new CheckstyleTool(buildOutput, 'checkstyleAnalysisEnabled'),
-        new PmdTool(buildOutput, 'pmdAnalysisEnabled')]);
+        new PmdTool(buildOutput, 'pmdAnalysisEnabled'),
```

Comment:
Not very important, but you might want to keep the order in which hte tools appear in the UI.